lib/registries/addon/components/registration-list/card/component.ts

```diff
@@ -0,0 +1,53 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+
+import RegistrationModel from 'ember-osf-web/models/registration';
+import ReviewActionModel from 'ember-osf-web/models/review-action';
+
+const iconMap: Record<string, string> = {
```

Comment:
Use `RegistrationReviewStates`
```suggestion
const iconMap: Record< RegistrationReviewStates, string> = {
```